frontend/class-public.php

```diff
@@ -308,20 +325,40 @@ public function hide_default_read_more( $more_text ){
 	 *
 	 * Displays the volunteer opportunities lists for both the one-time and flexible
 	 * opportunities. It also calls template files to output the majority of the HTML.
-	 * 
-	 * @param  string $list_type One-time or flexible volunteer opportunities
-	 * @param  array $query_args The query arguments to be used in WP_Query 
+	 *
+	 * @param  string $list_type One-time or flexible volunteer opportunities.
+	 * @param  array  $query_args The query arguments to be used in WP_Query.
+	 * @param array  $block_attributes  Attributes saved from the block editor to display in the admin or frontend.
 	 * @return string            HTML code to be output via a shortcode.
 	 */
-	public function display_volunteer_opp_list( $list_type, $query_args ){
-		//We must edit the main query in order to handle pagination.
+	public function display_volunteer_opp_list( $list_type, $query_args, $block_attributes ) {
+		// We must edit the main query in order to handle pagination.
 		global $wp_query;
 		$temp = $wp_query;
 		$wp_query = new WP_Query( $query_args );
 
-		ob_start(); ?>
-		
-		<div class="volunteer-opps <?php echo $list_type; ?>">
+		ob_start();
+
+		$class_name = $list_type;
+		$anchor     = '';
+
+		if ( is_array( $block_attributes ) ) {
+
+			// If the Additonal CSS class(es) field on the block is used, include it in the container class name.
+			if ( array_key_exists( 'className', $block_attributes ) && $block_attributes['className'] !== '' ) {
+
+				$class_name = $class_name . ' ' . sanitize_html_class( $block_attributes['className'] );
```

Comment:
It looks like `sanitize_html_class()` is removing the space if multiple classes are provided for the block.

Reply:
Ugh, thank you for catching this. I'll see if I can use a different WordPress sanitization method.

Reply:
Looks like sanitize_text_field() will work well here!